site/components/pages/Pledge/PledgeDashboard/Profile/index.tsx

```diff
@@ -46,24 +50,26 @@ export const Profile: FC<Props> = (props) => {
   const pledgeProgress =
     totalTonnesRetired && (totalTonnesRetired / currentFootprint.total) * 100;
   const displayPledgeProgress =
-    !isNaN(totalTonnesRetired) &&
-    !isNaN(totalTonnesRetired) &&
-    currentFootprint.total > 0;
+    !isNaN(totalTonnesRetired) && currentFootprint.total > 0;
 
   const renderPledgeProgress = () => {
     if (!displayPledgeProgress) return null;
 
     if (pledgeProgress >= 100) {
       return (
         <Text t="h4" className={styles.pledgeProgress}>
-          &ge; 100% of Pledge Met
+          <Trans id="pledges.dashboard.profile.pledge_met">
+            &ge; 100% of Pledge Met
```

Comment:
Interesting, why not use "≥". instead?

Reply:
Oh yeah good one, not sure why I didn't go for the simple approach 😆